### PR TITLE
Geoconda 3.10.9 updates

### PR DESCRIPTION
**Pull request by ktiits**
https://csc-guide-preview.rahtiapp.fi/origin/gis-geoconda-202302/apps/geoconda/

### CODE REVIEW

File: docs/apps/geoconda.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ includes following python packages:
 -   [boto3](https://boto3.readthedocs.io) - for working files in S3 storage, for example Allas. [Allas S3 example in CSC geocomputing Github](https://github.com/csc-training/geocomputing/blob/master/python/allas/working_with_allas_from_Python_S3.py).
 -   [cartopy] - for map plotting.
 -   [cfgrib](https://pypi.org/project/cfgrib/) - map GRIB files to the NetCDF Common Data Model
+-   [copc-lib](https://github.com/RockRobotic/copc-lib) - reader and writer interface for [COPC point clouds](https://copc.io/) NEW 2023
```
Comment (samumantha) [inline docs/apps/geoconda.md:15]:
```suggestion
-   [copc-lib](https://github.com/RockRobotic/copc-lib) - reader and writer interface for [Cloud Optimized Point Clouds (COPC)](https://copc.io/) NEW 2023
```

File: docs/apps/whiteboxtools.md [hunk 2]
```diff
@@ -5,20 +5,32 @@ tags:
 
 # WhiteboxTools
 
-[WhiteboxTools](https://www.whiteboxgeo.com/manual/wbt_book/intro.html) is an advanced geospatial data analysis platform which includes more than 480 tools. Many tools operate in parallel, taking full advantage of your multi-core processor.
+[WhiteboxTools](https://www.whiteboxgeo.com/manual/wbt_book/intro.html) is an advanced geospatial data analysis platform which includes more than 450 tools. Many tools operate in parallel, taking full advantage of your multi-core processor.
 
 ## Available
```
Comment (samumantha) [inline docs/apps/whiteboxtools.md:10]:
```suggestion
## Availability
```

File: docs/apps/geoconda.md [hunk 3]
```diff
@@ -74,9 +79,10 @@ If you think that some important GIS package for Python is missing from here, yo
 
 ## Available
```
Comment (samumantha) [inline docs/apps/geoconda.md:80]:
```suggestion
## Availability
```

File: docs/apps/whiteboxtools.md [hunk 4]
```diff
@@ -5,20 +5,32 @@ tags:
...
 
-WhiteboxTools is available in Puhti with following versions:
+Only WhiteboxTools Open Core tools are available on Puhti. WhiteboxTools is available with following versions:
 
-* 2.1.0. Only WhiteboxTools Open Core tools are available on Puhti.
+* 2.2.0 
+* 2.1.0
```
Comment (samumantha) [inline docs/apps/whiteboxtools.md:15]:
```suggestion
* 2.2.0 in the **geoconda** module
* 2.1.0 in the **WhiteboxTools** module 
```

File: docs/apps/whiteboxtools.md [hunk 5]
```diff
@@ -5,20 +5,32 @@ tags:
...
 
 ## Usage
 
-WhiteboxTools is available in the __WhiteboxTools__ module and can be loaded with
+WhiteboxTools is available
+
+* 2.2.0 in the **geoconda** module
+* 2.1.0 in the **WhiteboxTools** module 
+
```
Comment (samumantha) [inline docs/apps/whiteboxtools.md:23]:
```suggestion
```

File: docs/apps/geoconda.md (hunk 1 above)
Comment (samumantha) [reply docs/apps/geoconda.md:15]: Would write it out here as it is not widely known acronym (yet?)

**Review by samumantha — CHANGES_REQUESTED** [review]
Small suggestion to reorganize in whitebooxtools

File: docs/apps/geoconda.md (hunk 3 above)
Comment (ktiits) [reply docs/apps/geoconda.md:80]: Available is old heading used in all Apps pages, would not change it now.

**Review by samumantha — APPROVED** [review]

**Pull request merged**